src/views/which-type.njk

```diff
@@ -64,4 +64,16 @@
     </div>
   </div>
 
+  <script>
+    const radioButtons = document.querySelectorAll('input[name="parameter"]');
+
+    radioButtons.forEach(function(radioButton) {
+      radioButton.addEventListener('change', function() {
+        if (radioButton.checked) {
+          sessionStorage.setItem("which-type", radioButton.value);
```

Comment:
As we will end up doing 3 different journeys, we may need to use this solution again, would it be better to name the sessionStorage value something like register-which-type or registration-which-type to reduce chances of a clash?